src/types/index.ts

```diff
@@ -0,0 +1,188 @@
+/**
+ * Evaluation status for a criteria
+ *
+ * FRAMEWORK USAGE NOTES:
+ * - PASS: Criterion automatically verified as meeting requirements
+ * - FAIL: Criterion automatically verified as NOT meeting requirements
+ * - MANUAL: Requires human review (e.g., errors during evaluation, non-automatable criteria)
+ * - NOT_APPLICABLE: Evaluation logic not yet implemented (stub)
+ *
+ */
+export enum EvaluationStatus {
+  PASS = 'pass',
+  FAIL = 'fail',
+  MANUAL = 'manual',
+  NOT_APPLICABLE = 'not_applicable'
```

Comment:
> @maccabeelevine Currently most of the criteria report back MANUAL, but some of these are because they are stubs for future automation, and some because they will (presumably) always be manual. I wonder if we should split those two cases for clarity on what we intend. Although on the other hand, deciding now whether something can (or should) be automated in the future may be impossible.

I added a NA status and update the stubs to have this status.